.renovaterc.json

```diff
@@ -4,39 +4,46 @@
     ":enablePreCommit",
     "schedule:weekly"
   ],
-  "ignoreDeps": ["relekang/python-semantic-release"],
-  "pip_requirements": {
-    "fileMatch": ["^requirements(-[\\w]*)?\\.txt$"]
-  },
+  "enabledManagers": [
+    "pip_requirements",
+    "pre-commit"
+  ],
```

Comment:
Just a quick question, was this suggested by `renovate-config-validator`? IIRC `enabledManagers` is an allowlist and others will be ignored, so we might need to maintain this list manually (e.g. in https://github.com/python-gitlab/python-gitlab/blob/main/Dockerfile.

Reply:
It was not suggested by `renovate-config-validator`. Not sure I am following the rest of your comment.

Reply:
Hmmm. I am giving it another thought. It is better to remove this section and let precommit take into consideration all the managers.

Reply:
That's true, but I agree it's best to let renovate pick up any potential updates yes :) thanks.